include/llvm/ADT/STLExtras.h

```diff
@@ -18,7 +18,6 @@
 #define LLVM_ADT_STLEXTRAS_H
 
 #include "llvm/Support/Compiler.h"
-#include "llvm/Support/WinSAL.h"
 #include <algorithm> // for std::all_of
```

Comment:
This header and a few others include __cdecl, which is defined in the new WinAdaptor.h. I suppose it's included somewhere above? Or just always included first in the source files?